modules/friendlytag.js

```diff
@@ -679,88 +679,88 @@ Twinkle.tag.article = {};
+	"Autobiography": "autobiography and may not be written neutrally",
+	"BLP sources": "BLP that needs additional sources for verification",
+	"BLP unsourced": "BLP that has no sources at all (use BLP PROD instead for new articles)",
+	"Citation style": "unclear or inconsistent citation style",
+	"Cleanup": "requires cleanup",
+	"Cleanup bare URLs": "uses bare URLs for references, which are prone to link rot",
+	"Cleanup-PR": "reads like a press release or news article",
+	"Cleanup reorganize": "needs reorganization to comply with Wikipedia's layout guidelines",
+	"Cleanup rewrite": "needs to be rewritten entirely to comply with Wikipedia's quality standards",
+	"Cleanup tense": "does not follow guidelines on use of different tenses.",
+	"Close paraphrasing": "contains close paraphrasing of a non-free copyrighted source",
+	"COI": "creator or major contributor may have a conflict of interest",
+	"Condense": "has too many section headers dividing up its content",
+	"Confusing": "confusing or unclear",
+	"Context": "provides insufficient context for those unfamiliar with the subject",
+	"Copy edit": "requires copy editing for grammar, style, cohesion, tone, or spelling",
+	"Copypaste": "appears to have been copied and pasted from another location",
+	"Current": "documents a current event",
+	"Disputed": "questionable factual accuracy",
+	"Essay-like": "written like a personal reflection, personal essay, or argumentative essay",
+	"Expand language": "should be expanded with text translated from a foreign-language article",
+	"Expert needed": "needs attention from an expert on the subject",
+	"External links": "external links may not follow content policies or guidelines",
+	"Fanpov": "written from a fan's point of view",
+	"Fiction": "fails to distinguish between fact and fiction",
+	"Globalize": "may not represent a worldwide view of the subject",
+	"GOCEinuse": "currently undergoing a major copy edit by the Guild of Copy Editors",
 	"History merge": "another page should be history merged into this one",
-	"Hoax": "article may be a complete hoax",
-	"Improve categories": "article may require additional categories",
-	"Incomprehensible": "article is very hard to understand or incomprehensible",
-	"In-universe": "article subject is fictional and needs rewriting from a non-fictional perspective",
-	"In use": "article is undergoing a major edit for a short while",
-	"Lead missing": "article has no lead section and one should be written",
-	"Lead rewrite": "article lead section needs to be rewritten to comply with guidelines",
-	"Lead too long": "article lead section is too long and should be shortened",
-	"Lead too short": "article lead section is too short and should be expanded",
-	"Like resume": "article is written like a resume",
-	"Long plot": "plot summary in article is too long",
-	"Manual": "article is written like a manual or guidebook",
-	"Merge": "article should be merged with another given article",
+	"Hoax": "may be a complete hoax",
```

Comment:
```suggestion
	"Hoax": "may partially or completely be a hoax",
```
more accurate, per template